drop_cache_if_idle

```diff
@@ -78,11 +78,11 @@ def main():
     if load_check():
         is_apply_succeed = apply()
 
-    # Let's give the proper output to the user in case of failure
-    if is_apply_succeed:
-        print("INFO: The command is executed successfully.")
-    else:
-        print("INFO: There was an error preventing the command being executed.")
+        # Let's give the proper output to the user in case of failure
+        if is_apply_succeed:
+            print("INFO: The command is executed successfully.")
```

Comment:
Yes, I agree with you. But also I don't think that we should have two "executed successfully" messages if it is executed. Hence, I think we should drop `print("INFO: The command is executed successfully.")`  on line 83 and make the if statement as `if not is_apply_succeed:` (or equivalent). This is my first time using the GitHub review tool, sorry if I made the review in the wrong way. Thanks for the contribution!

Reply:
Makes sense to me. I'll update mine accordingly.